Intersect.Client.Framework/Gwen/Control/RichLabel.cs

```diff
@@ -67,6 +67,22 @@ public override JObject GetJson()
             return base.FixJson(obj);
         }
 
+        /// <summary>
+        ///     Adds text to the control via the properties of a <see cref="Label"/>
+        /// </summary>
+        /// <param name="text">Text to add</param>
+        /// <param name="template">Label to use as a template</param>
+        public void AddText(string text, Label template)
+        {
+            if (template == null)
+            {
+                throw new ArgumentNullException(nameof(template));
+            }
+            AddText(text, template.TextColor,
+                template.CurAlignments.Count > 0 ? template.CurAlignments[0] : Alignments.Left,
```

Comment:
I'm not sure but would like to see if `template.CurAlignments.FirstOrDefault() ?? Alignments.Left` works -- it may not because `Alignments` is an enum.

Reply:
No dice :P